### PR TITLE
feat(archetype-service-rest-reactive-post): add archetype for post rea…

### PR DESCRIPTION
**Pull request by eaquin02**
…ctive rest service

This PR is for adding the synapse-archetype-service-rest-reactive-post module.

### CODE REVIEW

**Review by whoswendy — CHANGES_REQUESTED** [review]
Please add dependency to main pom.xml and add __className__Application.java

File: archetype/synapse-archetype-service-rest-reactive-post/src/main/resources/archetype-resources/src/main/java/__className__Application.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    /**
+     * Main method to run the Spring Boot ${className} Application.
+     *
+     * @param args the args
+     */
+    public static void main(String[] args) {
+        SpringApplication.run(${className}Application.class, args);
+        LOGGER.info("${className} Application is up and running ...");
+    }
+}
```
Comment (whoswendy) [inline archetype/synapse-archetype-service-rest-reactive-post/src/main/resources/archetype-resources/src/main/java/__className__Application.java]: add new line eof please

**Review by whoswendy — APPROVED** [review]

File: archetype/synapse-archetype-service-rest-reactive-post/src/main/resources/archetype-resources/src/main/resources/service-application.properties [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+spring.application.name=${apiName} Reactive Create Rest Service
+spring.main.allow-bean-definition-overriding=true
+server.port=8080
+wavefront.application.name=${apiName}
+wavefront.application.service=${apiName}-reactive-create-rest-service
```
Comment (guttff) [inline archetype/synapse-archetype-service-rest-reactive-post/src/main/resources/archetype-resources/src/main/resources/service-application.properties:16]: Can the user enter ${apiName} with any casing?

Comment (eaquin02) [reply archetype/synapse-archetype-service-rest-reactive-post/src/main/resources/archetype-resources/src/main/resources/service-application.properties:16]: I tested it and wavefront still works with any casing

**Review by guttff — APPROVED** [review]

**Pull request merged**